src/base/io-funcs-test.cc

```diff
@@ -45,6 +45,16 @@ void UnitTestIo(bool binary) {
     std::vector<char> vec3;
     for (size_t i = 0; i < 10; i++) vec3.push_back(Rand()%100);
     WriteIntegerVector(outfile, binary, vec3);
+    std::vector<std::pair<int32, int32> > vec4;
```

Comment:
best not to leave the sizes fixed at 10 but make them random.  you might hit a special case somewhere (e.g. size = 1).